.github/workflows/snyk.yml

```diff
@@ -0,0 +1,55 @@
+name: Snyk SBOM CI
```

Comment:
nit: could you please rename this to match the other CI steps that get run, e.g. `snyk-sbom-ci`